### PR TITLE
SC-Designing-fix

### PR DESCRIPTION
**Pull request by shital131**
Change the Logout button and Background of a header.

### CODE REVIEW

**Review by shital131 — COMMENTED** [review]
Changes the Logout Button with a Circle within a image add View Profile, SignIn option in a list. And change the background color of a menu bar.

**Comment by shital131** [comment]
Revert changes in some files.

**Comment by shital131** [comment]
Add js in my code and revert  changes in header component.

File: src/index.html [hunk 1]
```diff
@@ -9,8 +9,32 @@
...
   <!-- <link href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" /> -->
+  
 </head>
 <body>
   <app-root></app-root>
+
+  <script>
+
+        /* When the user clicks on the button,
+     toggle between hiding and showing the circle content */
```
Comment (kunwarkundanbblock) [inline src/index.html:20]: Remove this script from here.

File: src/app/views/user-management/user-management.component.html [hunk 2]
```diff
@@ -24,14 +24,14 @@
       <div class="box header">Phone Number</div>
       <div class="box header">Batch</div>
     </div>
-    <div *ngFor="let user of userList" class="row">
+    <!-- <div *ngFor="let user of userList" class="row">
```
Comment (kunwarkundanbblock) [inline src/app/views/user-management/user-management.component.html:27]: uncomment these lines.

**Review by kunwarkundanbblock — CHANGES_REQUESTED** [review]
Uncomment the lines in user html

**Comment by shital131** [comment]
Done.

**Pull request merged**